handbook/marketing/content/editorial.md

```diff
@@ -38,7 +38,8 @@ The secondary function of the blog is to grow awareness of Sourcegraph’s brand
 
 The buckets outlined below are an internal reference and guideline. Although there are three buckets here, content we may publish isn't limited to these, and we may publish more of one type of content than others.
 
-1. **Inside Sourcegraph engineering:** How we built X, scaled Y, debugged Z. These are meaty deep dives into the process of building and iterating on Sourcegraph the product and engineering org. Got code snippets? Graphs and charts? Links to PRs? We’ll take them all. Even though we may talk about Sourcegraph as part of these stories, the focus is on the technical work behind the product rather than the product itself.
+1. **Inside Sourcegraph engineering:** How we built X, scaled Y, debugged Z. These are meaty deep dives into the process of building and iterating on Sourcegraph the product and engineering org. Got code snippets? Graphs and charts? Links to PRs? We’ll take them all. Even though we may talk about Sourcegraph as part of these stories, the focus is on the technical work behind the product rather than the product itself. Please note that these posts should be written from the company (we/us) perspective (see Better onboarding: [How to prevent codebase overwhelm](https://about.sourcegraph.com/blog/better-onboarding-how-to-prevent-codebase-overwhelm/) rather than a writer/reporter perspective.
```

Comment:
@nickmyyz Thank you so much for getting this into our handbook! 🙌 

Given that you recommended we combine the onboarding posts, we won't be able to use this example much longer. Could you use an alternative?

Also, can you delete the blank line underneath (line 42) as the numbering doesn't work when there are gaps between the items in the numbered list.